en/docs/authentication-and-authorization/secure-communication-between-the-choreo-gateway-and-your-backend.md

```diff
@@ -0,0 +1,130 @@
+# Secure Communication Between the Choreo Gateway and Your Backend
+
+To establish secure communication between the Choreo Gateway and your backend, you can configure mutual SSL.
+
+Mutual SSL authentication involves both the client and server validating each other’s certificates before establishing a connection. The following diagram depicts this scenario:
+
+![Mutual SSL authentication](../assets/img/authentication-and-authorization/mutual-ssl-authentication.png)
+
+## Configure mutual SSL to establish secure connectivity
+
+To establish secure connectivity between the Choreo Gateway and your backend using mutual SSL, you must add the certificate of the backend (server certificate) to Choreo and add the certificate of Choreo (client certificate) as a trusted certificate in the backend.
+
+### Step 1: Configure TLS for a self-signed certificate
```

Comment:
What we need to mention here is about setting up backend certificate right? Should we change the title accordingly?
`Configure backend service certificate`